schema/musicxml.xsd

```diff
@@ -457,6 +457,18 @@ The values of start, stop, and continue refer to how an element appears in music
 		</xs:restriction>
 	</xs:simpleType>
 
+	<xs:simpleType name="tied-type">
+		<xs:annotation>
+			<xs:documentation>The tied-type type is used as an attribute of the tied element to specify where the visual representation of a tie begins and ends. A tied element which joins two notes of the same pitch can be specified with tied-type start on the first note and tied-type stop on the second note. To indicate a note should be undamped, use a single tied element with tied-type let-ring. For other ties that are visually attached to a single note, such as a tie leading into or out of a repeated section or coda, use two tied elements on the same note, one start and one stop.</xs:documentation>

```

Comment:
Same comments as above about removing "of the same pitch" and about mentioning the continue attribute value.